tests/integration/test_dataapi.py

```diff
@@ -18,7 +18,9 @@
 items_request = {
     # These items should already exist for the demos consumer
     'references': ['item_2', 'item_3'],
-    'limit': 1
+    'limit': 1,
+    # Add a name that does not contain special characters
+    'name': 'Tect'
```

Comment:
`name` is not a valid parameter for `GET items` https://reference.learnosity.com/data-api/endpoints/itembank_endpoints#getItems, so there's no guarantee that 
1. the signature spans that (it does, for now, but not sure in the future),
2. api-data would accept it without error (ditto).

I'd suggest not doing anything here, and `appending` a special-charactered reference to `items_request['references']` in `test_real_request_with_special_characters`.

Reply:
Good shout.